src/styles/blocks/container.scss

```diff
@@ -0,0 +1,19 @@
+.container {
```

Comment:
```suggestion
.container { 
```

try to use min width to limit the container's width overall on narrow screens ..use 1024 px